.gitignore

```diff
@@ -97,6 +98,9 @@ ENV/
 # Rope project settings
 .ropeproject
 
+# Visual Code settings
```

Comment:
I would recommend adding personal editor to the global gitignore so we don't end up adding one ignore by editor depending on the personal taste of each one.

Reply:
👍